### PR TITLE
Recommend Yarn for installation

### LINKED ISSUE #1129: Recommend Yarn for installation
In discussion regarding how to decrease installation time, @sindresorhus suggested recommending Yarn as a temporary solution for this problem. Installing AVA with Yarn is a great experience, so we might as well recommend it officially.

This involves putting a simple note in readme under [installation](https://github.com/avajs/ava#add-ava-to-your-project) section. Don't forget to link to Yarn too ;)

### PR DESCRIPTION
**Pull request by anoff**
Fixes #1129 
Also bumped the expected version in the `package.json`

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -64,10 +64,11 @@ test(t => {
 
 ### Add AVA to your project
 
-Install AVA globally and run it with `--init` to add AVA to your `package.json`:
+Install AVA globally and run it with `--init` to add AVA to your `package.json`.
+We recommend using [yarn](https://yarnpkg.com/) to speed up the installation process.
```
Comment (sindresorhus) [inline readme.md]: It should say that Yarn is a better alternative to npm.

File: readme.md [hunk 2]
```diff
@@ -64,10 +64,11 @@ test(t => {
...
 
 ```console
-$ npm install --global ava
+$ yarn global add ava
```
Comment (sindresorhus) [inline readme.md]: It should not replace `npm`, but be a recommended alternative.

Like:

We recommend using Yarn bla bla.

yarn install

Alternatively with npm:

npm install

File: readme.md [hunk 3]
```diff
@@ -80,7 +81,7 @@ Your `package.json` will then look like this:
     "test": "ava"
   },
   "devDependencies": {
-    "ava": "^0.15.0"
+    "ava": "^0.17.0"
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.

File: readme.md [hunk 4]
```diff
@@ -92,7 +93,7 @@ Any arguments passed after `--init` are added as config to `package.json`.
 You can also install AVA directly:
 
 ```console
-$ npm install --save-dev ava
+$ yarn add --dev ava
```
Comment (sindresorhus) [inline readme.md]: Same as above.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: yarn => Yarn

File: readme.md [hunk 5]
```diff
@@ -64,9 +64,20 @@ test(t => {
 
 ### Add AVA to your project
 
-Install AVA globally and run it with `--init` to add AVA to your `package.json`:
+Install AVA globally and run it with `--init` to add AVA to your `package.json`.
+[Yarn](https://yarnpkg.com/) currently provides significant speed improvements
+over npm during the installation process. Consider using [Yarn](https://yarnpkg.com/en/docs/install)
+if the installation is too slow for your needs.
```
Comment (sindresorhus) [inline readme.md]: Don't hard-wrap.

File: readme.md [hunk 6]
```diff
@@ -64,9 +64,20 @@ test(t => {
 
 ### Add AVA to your project
 
-Install AVA globally and run it with `--init` to add AVA to your `package.json`:
+Install AVA globally and run it with `--init` to add AVA to your `package.json`.
+[Yarn](https://yarnpkg.com/) currently provides significant speed improvements
+over npm during the installation process. Consider using [Yarn](https://yarnpkg.com/en/docs/install)
```
Comment (sindresorhus) [inline readme.md]: `using [Yarn]` => `[using Yarn]`

File: readme.md [hunk 7]
```diff
@@ -92,6 +103,12 @@ Any arguments passed after `--init` are added as config to `package.json`.
 You can also install AVA directly:
 
 ```console
+$ yarn add --dev ava
+```
+
+alternatively using npm:
```
Comment (sindresorhus) [inline readme.md]: `alternatively` => `Alternatively`

**Comment by anoff** [comment]
appveyor build seemed to have run into a timeout during npm installation. Not sure what I did to break that?

**Comment by sindresorhus** [comment]
Very good. Thank you @anoff. Keep the PRs coming 😎👍
